branchinfo.txt

```diff
@@ -6,6 +6,6 @@ MINOR_VERSION=1
 PATCH_VERSION=0
 STABILIZE_PACKAGE_VERSION=false
 LOCK_HOST_VERSION=false
-RELEASE_SUFFIX=preview1
+RELEASE_SUFFIX=preview2
```

Comment:
I'm not sure this file is used anymore but probably still worth updating until it gets deleted.